tests/test_live_validation.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * This file is part of RTRlib.
```

Comment:
I suggest that all new code has to comply with the kernel coding style. Otherwise we have to reformat it again.
All new files should be verified with checkpatch.pl and added to https://github.com/rtrlib/rtrlib/blob/lpfst_cleanup/scripts/check-coding-style.sh to be checked.